hummingbird/ml/ir_converters/linked_node.py

```diff
@@ -0,0 +1,314 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Converters for LinkedNode IR are stored in this file.
+"""
+
+import os
+from uuid import uuid4
+from distutils.version import LooseVersion
+
+from onnxconverter_common.optimizer import LinkedNode, _topological_sort
+from onnxconverter_common.registration import get_converter
+import onnxruntime as ort
+import onnx
+from onnx import helper, shape_inference
+import torch
+
+from ..operator_converters import constants
+from ..supported import get_onnxml_api_operator_name
+from ..exceptions import MissingConverter
+
+
+def convert(
+    node_list, input_tensors, initializers, output_names, test_data, output_model_name=None, target_opset=9, extra_config={},
```

Comment:
Why 9 and not 11?  (is there some particular operator we use?)

Reply:
9 is default in the torch to onnx converter.